src/actions/index.js

```diff
@@ -0,0 +1,18 @@
+let nextBookId = 0
+
+export const createBook = ({title, category}) => ({
+  type: 'CREATE_BOOK',
+  id: nextBookId++,
+  title,
+  category
+});
+
+export const removeBook = ({id}) => ({

```

Comment:
I wrap it because I destructure it form the book object which is passed as a parameter. I am not sure about passing the book instead of just an id, but the project instructions say this:
![image](https://user-images.githubusercontent.com/9296248/61385862-dfe22000-a8bb-11e9-9b99-e70f599cef52.png)